gradle.properties

```diff
@@ -5,7 +5,7 @@ kotlin.mpp.stability.nowarn=true
 kotlin.native.ignoreDisabledTargets=true
 
 # SDK
-sdkVersion=0.27.8-SNAPSHOT
+sdkVersion=0.28.0-SNAPSHOT
```

Comment:
**Question**: Why did this necessitate a major version bump?

Reply:
It didn't, it fixes a previous commit where it should have been bumped: https://github.com/awslabs/smithy-kotlin/commit/681379d3cdc6ef578ae3b03b770dda1fc4a35aa0

Reply:
I was expecting the nullability PR to land with everything else which bumped the version, that won't land until next week (hopefully with the name changes which would have also bumped it)

Reply:
Blerg, it's causing downstream CI issues though so I'm reverting. @lauzadis we just need to remember to bump this before release.